x86_64-linux-gnu-binutils/PKGBUILD

```diff
@@ -0,0 +1,72 @@
+# Contributor: Levi Zim <rsworktech@outlook.com>
+# Contributor: Anatol Pomozov <anatol.pomozov@gmail.com>
+
+_target=x86_64-linux-gnu
+pkgname=$_target-binutils
+pkgver=2.43
+pkgrel=1
+pkgdesc='A set of programs to assemble and manipulate binary and object files for the ARM64 target'
+arch=(riscv64)
+url='https://www.gnu.org/software/binutils/'
+license=(GPL)
+depends=(zlib libelf)
+source=(https://ftp.gnu.org/gnu/binutils/binutils-$pkgver.tar.bz2{,.sig})
+sha1sums=('9b0b465bf52fdbb677bd67ba64424842e297ebcd'
+          'SKIP')
+sha256sums=('fed3c3077f0df7a4a1aa47b080b8c53277593ccbb4e5e78b73ffb4e3f265e750'
+            'SKIP')
+validpgpkeys=('3A24BC1E8FB409FA9F14371813FCEF89DD9E3C4F') # Nick Clifton (Chief Binutils Maintainer) <nickc@redhat.com>
+
+prepare() {
+  cd binutils-$pkgver
+  sed -i "/ac_cpp=/s/\$CPPFLAGS/\$CPPFLAGS -O2/" libiberty/configure
+}
+
+build() {
+  cd binutils-$pkgver
+
+  if [ "${CARCH}" != "i686" ];
+  then
+    # enabling gold linker at i686 makes the install fail
+    enable_gold='--enable-gold'
+  fi
```

Comment:
May consider dropping the branch?

Reply:
Thanks for the advice. I have dropped it.